plugin/modify-env/modify-env.c

```diff
@@ -48,7 +49,16 @@ void dmtcp_event_hook(DmtcpEvent_t event, DmtcpEventData_t *data)
   switch (event) {
   case DMTCP_EVENT_RESTART:
   { int size = 12288;
-    char *buf = read_dmtcp_env_file("dmtcp_env.txt", size);
+    int retval;
+    char env_file[PATH_MAX];
+
+    retval = dmtcp_get_restart_env("DMTCP_ENV_FILE", env_file, PATH_MAX);
+    if (retval != 0)
+    {
+        strcpy(env_file, "dmtcp_env.txt"); 
```

Comment:
Thought in this case, since we were copying hard-coded "dmtcp_env.txt" that there is low risk here.  But if we add the macros as below, this suggests someone might change it in the future.  I'll rework.